src/content/overview/observability/configuration/_index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Configuration
+description: Learn how to configure and customize the Giant Swarm observability platform for your organization's needs.
+weight: 30
+menu:
+  principal:
+    parent: overview-observability
+    identifier: overview-observability-configuration
+last_review_date: 2025-06-30
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - How do I configure the observability platform?
+  - What configuration options are available?
+  - How do I set up multi-tenancy?
+  - What can I customize in Grafana?
+  - How do I configure data ingestion?
+  - What alerting options can I configure?
+  - How do I manage access controls?
+---
+
+The Giant Swarm observability platform offers extensive configuration options to tailor the system to your organization's specific monitoring, alerting, and visualization needs. This overview covers the key areas you can configure to create a customized observability experience that fits your operational requirements.
+
+## Platform-level configuration
+
+Configure foundational platform settings that affect the entire observability stack:
+
+### Initial platform setup
+
+Work with Giant Swarm to configure core platform components:
+
+- **OIDC provider integration**: Connect your identity provider for authentication
+- **Base domain configuration**: Set up DNS and ingress for platform access
```

Comment:
I am a bit lost. In the OIDC provider configuration, don't we have a specific page for it? Can we link it? And based on the domain configuration, it depends on whether it is cloud or on-prem, right? 

Is this thought to be ready for non GS environments?

Reply:
I could not find specific pages about those 2 so I'm honestly not sure.

It it not ready for non GS env because we do not have any today

Reply:
Should we remove this?

Reply:
We have this page we could link: https://docs.giantswarm.io/tutorials/access-management/configure-dex-in-your-cluster/ - that might be just the helpful page here, right?

Reply:
But I don't think we have a page for the general base domain configuration :/

Reply:
But this is mostly about configuring Dex on WCs right? Because configuring Dex on MCs is not in self service?

Reply:
ok let's leave it like it is and we can always link when we add proper docs